app/src/main/kotlin/com/saltedge/authenticator/features/connections/list/ConnectionsListViewModel.kt

```diff
@@ -104,11 +121,15 @@ class ConnectionsListViewModel(
             }
             PopupMenuItem.LOCATION -> onAccessToLocationClickEvent.postUnitEvent()
             PopupMenuItem.DELETE -> {
-                if (item.isActive) {
-                    onDeleteClickEvent.postValue(ViewModelEvent(item.guid))
+                if (!noInternetConnection) {
+                    if (item.isActive) {
+                        onDeleteClickEvent.postValue(ViewModelEvent(item.guid))
+                    } else {
+                        interactor.revokeConnection(item.guid)
```

Comment:
I don't see changes in interactor. I think you should handle REST requests errors (NoConnection, etc).